pkgs/servers/monitoring/longview/default.nix

```diff
@@ -0,0 +1,51 @@
+{stdenv, fetchFromGitHub, perl, perlPackages, makeWrapper, glibc }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.5pre";
+  name = "longview-${version}";
+
+  src = fetchFromGitHub {
+    owner = "linode";
+    repo = "longview";
+    rev = "5bcc9b60896b72de2d14f046f911477c26eb70ba";
+    sha256 = "1i6va44bx2zfgbld7znf1slph0iqidlahq2xh3kd8q4lhvbrjn02";
+  };
+
+  patches =
+    [ # Allow the paths of the config files to be set at runtime
+      ./runtime-paths.patch
```

Comment:
would be nice if you submited this patch upstream and added a link to the bug here